Capstone/Authentication/RegistrationViewController.m

```diff
@@ -23,6 +23,7 @@ - (void)registerUser {
     newUser.username = self.usernameRegistrationField.text;
     newUser.email = self.emailRegistration.text;
     newUser.password = self.passwordRegistration.text;
+    newUser[@"Language"] = @"English";
```

Comment:
if default is english, as sophia mentioned, do we need to include this?

Reply:
I was trying our another method before hand and forgot to remove it. I agree that it is not needed :)